segments/helpers.py

```diff
@@ -159,7 +159,7 @@ def execute_raw_user_query(sql):
 
         # Guardrail: Try to ensure results are integers
         if total > 0 and result[0]:
-            if not type(result[0][0]) == int:
+            if not all(isinstance(i, int) for i in result[0]):
```

Comment:
Since you mentioned that this can be a UUID as well, and the bug occurs if a null value is returned, can't we check that the query result is not null instead?

Reply:
@apolisskaya Thanks for comment, will do. We also might want to just "skip" None values, since we would not be able to identify the user, instead of raising an exception, which prevents the segment from refreshing. My point is that it's better to silently skip "None" users, what do you think ?

Reply:
I think that would work as well! Your call :)